build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -51,6 +51,11 @@ export type FluidPackageJson = {
 	 * fluid-build config. Some properties only apply when set in the root or release group root package.json.
 	 */
 	fluidBuild?: IFluidBuildConfig;
+
+	/**
+	 * pnpm config
+	 */
+	pnpm?: any;
```

Comment:
Can we type this as either `unknown` or add a bit more of the structure?

Reply:
Adding `overrides` as an optional Record<string, string>. I debated adding `peerDependencyRules` which seems to be the other "official" one we use (`comments` an such I don't think make much sense here), but I think keeping it minimal and update if we ever need the other bits seems prefereable.